### PR TITLE
Crop kspace transforms

### PR DESCRIPTION
**Pull request by georgeyiasemis**
Changes:
* `CropKspace` transform implemented: Transforms to the image domain all coil data, crops to shape, transforms back to kspace domain
* `CropAndMask` transform removed (had no cropping functionality)
* `ComputeImage` transfrom implemented: computes image based on kspace
* `ApplyMask` transform implemented
* `build_mri_transforms` reformatted to abide to the new transforms
* Merged from main (https://github.com/NKI-AI/direct/pull/210/commits/446326846b3a84858f4e1f190686536895bfbd8d)

### CODE REVIEW

File: direct/nn/mri_models.py [hunk 1]
```diff
@@ -141,7 +141,12 @@ def l1_loss(source: torch.Tensor, reduction: str = "mean", **data) -> torch.Tens
                 L1 loss.
             """
             resolution = get_resolution(**data)
-            l1_loss = F.l1_loss(*_crop_volume(source, data["target"], resolution), reduction=reduction)
+            l1_loss = F.l1_loss(
+                *_crop_volume(
+                    T.modulus_if_complex(source, complex_axis=self._complex_dim), data["target"], resolution
+                ),
+                reduction=reduction,
+            )
```
Comment (jonasteuwen) [inline direct/nn/mri_models.py:149]: Have always been thinking about being a bit more flexible here. Perhaps we can look into making it configurable and allow for complex loss?

**Review by jonasteuwen — CHANGES_REQUESTED** [review]
Some small comments. Nothing needs to change in the configs I presume? Did you try training some models to check performance against older models?

File: direct/nn/mri_models.py (hunk 1 above)
Comment (georgeyiasemis) [reply direct/nn/mri_models.py:149]: Yes sure. But on a new issue/PR

File: setup.cfg [hunk 2]
```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 1.0.2
```
[the remark this answers is not in the document]
Comment (georgeyiasemis) [reply setup.cfg:2]: This was merged from main

File: direct/__init__.py [hunk 3]
```diff
@@ -2,4 +2,4 @@
 # Copyright (c) DIRECT Contributors
 
 __author__ = """direct contributors"""
-__version__ = "1.0.2"
```
[the remark this answers is not in the document]
Comment (georgeyiasemis) [reply direct/__init__.py:5]: This was merged from main
